Website/Import/WpImportManager.cs

```diff
@@ -8,11 +8,48 @@
 using Sitecore.Modules.WeBlog.Managers;
 using System.Xml;
 using System;
+using Sitecore.Resources.Media;
 
 namespace Sitecore.Modules.WeBlog.Import
 {
     public static class WpImportManager
     {
+        public static List<WpPost> Import(ID fileLocation, bool includeComments, bool includeCategories, bool includeTags)
+        {
+            Database master = Configuration.Factory.GetDatabase("master");

```

Comment:
We should be passing in the DB or DB name, not using a hard coded DB. Probably pass the DB in the constructor when making this class non-static.